ideascube/mediacenter/management/commands/remove_medias.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor nit: the command does not remove media, it removes leftover files.

So maybe the command (and thus the file) name should reflect that?